instrumentation/othttp/handler.go

```diff
@@ -79,6 +89,21 @@ func (h *Handler) configure(c *Config) {
 	h.spanNameFormatter = c.SpanNameFormatter
 }
 
+func (h *Handler) createMeasures() {
+	h.counters = make(map[string]*metric.Int64Counter)
+	h.valueRecorders = make(map[string]*metric.Int64ValueRecorder)
+
+	requestCounter := h.meter.NewInt64Counter(RequestCount)
+	requestBytesCounter := h.meter.NewInt64Counter(RequestBytes)
+	responseBytesCounter := h.meter.NewInt64Counter(ResponseBytes)
+	serverLatencyMeasure := h.meter.NewInt64ValueRecorder(ServerLatency)
```

Comment:
Should this be a `Float64ValueRecorder` instead?  I've found that integer milliseconds can be too coarse to actually capture latency information for some systems.

Reply:
Changed to microseconds